src/herald/utils.py

```diff
@@ -0,0 +1,29 @@
+# utils.py
+#
+# Copyright (C) 2019 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Utility functions for the project.
+
+
+from kano.utils.shell import run_cmd, run_cmd_log
+
+
+def get_logged_in_users():
+    """Get a set of currently logged in users."""
+
+    out, err, rc = run_cmd("who --users | awk '{print $1}'")
```

Comment:
missing filter for `tty7` which is where the desktop runs; interactive user - coupled with my comment above.

Reply:
I don't agree with this filter as it seems to hardcode something that is entirely specific to our particular environment. As is, the command returns all users who are logged in to the system irrespective of the means. The function then returns a `set` of all usernames to remove duplicates.

Reply:
Again Radu... who do we want to kano-sync? everyone? or just the only user who is logged in interactively?